x/profiles/keeper/keeper.go

```diff
@@ -189,3 +189,28 @@ func (k Keeper) DeleteAllDTagTransferRequests(ctx sdk.Context, user sdk.AccAddre
 	key := types.DtagTransferRequestStoreKey(user)
 	store.Delete(key)
 }
+
+func (k Keeper) DeleteDTagTransferRequest(ctx sdk.Context, owner, sender sdk.AccAddress, deletionType string) error {
+	var requests []types.DTagTransferRequest
+	store := ctx.KVStore(k.StoreKey)
+	key := types.DtagTransferRequestStoreKey(owner)
+	k.Cdc.MustUnmarshalBinaryBare(store.Get(key), &requests)
+
+	if len(requests) == 0 {
+		return fmt.Errorf("no requests to be %s", deletionType)
```

Comment:
I would avoid passing a `deletionType` parameter that is only used for logging. We can simply use the *delete* verb instead.